### PR TITLE
When Hot Reload change fails to apply in the browser send full error …

### PR DESCRIPTION
**Pull request by tmat**
…message to the client so that it can be logged

### CODE REVIEW

File: src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ setTimeout(async function () {
       const payload = JSON.parse(message.data);
       const action = {
         'UpdateStaticFile': () => updateStaticFile(payload.path),
-        'BlazorHotReloadDeltav1': () => applyBlazorDeltas(payload.sharedSecret, payload.deltas),
+        'BlazorHotReloadDeltav1': () => applyBlazorDeltas(payload.sharedSecret, payload.deltas, false),
+        'BlazorHotReloadDeltav2': () => applyBlazorDeltas(payload.sharedSecret, payload.deltas, true),
```
Comment (tmat) [inline src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js:47]: I preserved backward compat here, but I'm not sure what the versioning requirements actually here. Can this script be used across multiple versions of dotnet-watch? If not, why have version in the action name at all?

Comment (tlmii) [reply src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js:47]: I had a similar thought. I guess I could imagine a scenario where the injection occurs from one version of the SDK and dotnet-watch is run from another (particularly with local copies of the SDK like we have with arcade)? But it seems like you'd have to jump through some hoops to make that happen.

**Review by tlmii — APPROVED** [review]
LGTM

**Pull request merged**